Geometry/HcalCommonData/src/HcalSimParametersFromDD.cc

```diff
@@ -146,10 +146,11 @@ bool HcalSimParametersFromDD::build(const cms::DDCompactView* cpv, HcalSimulatio
   fv2.mergedSpecifics(ref2);
 
   while (fv2.firstChild()) {
-    const std::string name{fv2.name().data(), fv2.name().size()};
     const std::string matName{cms::dd::noNamespace(fv2.materialName()).data(),
                               cms::dd::noNamespace(fv2.materialName()).size()};
-    if (!isItHF(name, php) &&
+    std::vector<int> copy = fv2.copyNos();
+    int idet = (copy.size() > 1) ? (copy[1] / 1000) : 0;
+    if (((idet == 3) || (idet == 4)) &&
```

Comment:
Could you please use constants for 3 and 4? Or at least add a comment that explains what they are?

Reply:
added a comment line